lib/internal/Magento/Framework/App/Request/Http.php

```diff
@@ -311,13 +305,9 @@ public function getBeforeForwardInfo($name = null)
      */
     public function isAjax()
     {
-        if ($this->isXmlHttpRequest()) {
-            return true;
-        }
-        if ($this->getParam('ajax') || $this->getParam('isAjax')) {
-            return true;
-        }
-        return false;
+        return $this->isXmlHttpRequest()
+            || $this->getParam('ajax', null)
+            || $this->getParam('isAjax', null);
```

Comment:
The default value is `null`, you do not need to provide it explicitly

`\Magento\Framework\App\Request\Http\Proxy::getParam($key, $default = null)`